nservicebus/sqlserver/concurrency_content_sqlserver_[2.1,3).partial.md

```diff
@@ -0,0 +1,6 @@
+SQL Server transport uses an adaptive concurrency model. The transport adapts the number of polling threads based on the rate of messages coming in. The key concept in this new model is the *ramp up controller* which controls the ramping up of new threads and decommissioning of unnecessary threads. It uses the following algorithm:
+
+ * if last receive operation yielded a message, it increments the *consecutive successes* counter and resets the *consecutive failures* counter
```

Comment:
Also maybe we could just say something more generic, like that the algorithm adjusts the number of threads to the number of messages received in the given period of time (e.g. the higher the avg number of msmgs the higher the number of threads), without giving away all the details ;)

And say that it'll be never higher than `MaximumConcurrencyLevel`.